README.md

```diff
@@ -2,9 +2,9 @@
 
 [![Build Status](https://api.travis-ci.org/googleapis/gax-php.svg?branch=master)](https://travis-ci.org/googleapis/gax-php)
 
-[![Code Coverage](https://img.shields.io/codecov/c/github/googleapis/gax-php.svg)](https://codecov.io/github/googleapis/gax-php)
+[![Code Coverage](https://img.shields.io/codecov/c/github/googleapis/gax-php.svg?branch=master)](https://codecov.io/github/googleapis/gax-php?branch=master)
 
--   [Documentation](http://googleapis.github.io/gax-php)
+-   [Documentation](http://googleapis.github.io/gax-php/master/)
```

Comment:
The documentation link redirects to the latest tag, which is better than `master`.